src/SFA.DAS.EAS.Portal.Types/Account.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+
+namespace SFA.DAS.EAS.Portal.Types
+{
+    public class Account
+    {
+        [JsonConstructor]
+        public Account()
+        {
+            Organisations = new List<Organisation>();
+        }
+
+        [JsonProperty("id")]
+        public long Id { get; set; }
+        [JsonProperty("name")]
+        public string Name { get; set; }
+        [JsonProperty("organisations")]
+        public ICollection<Organisation> Organisations { get; set; }
+        [JsonProperty("deleted")]
+        public DateTime? Deleted { get; set; }
```

Comment:
Do we need `Deleted` at the `Account` level as well as at the `Document` level?

Reply:
I don't know.  There is no deleted requirement at the moment so maybe is should be removed until we have that requirement.

Reply:
Thinking about this, we'll probably need created, updated and deleted at more fine grained levels than just at the document/account level. We might need those attributes to have been set separately for the new event this PR adds (when we later add delete events etc.), along with the values for the existing event the worker handles.

We can give this more thought, but I think the safest thing to do would be to store the attributes separately for the `CohortApprovalRequestedByProvider` event and the `ReservationCreatedEvent`.

Reply:
We shouldn't try to guess future requirements.  If there is no requirement then we should remove it.

Reply:
It'd be nice if we could ignore future requirements, but unfortunately we need to think them through. We may find ourselves in a pickle if we haven't captured data or metadata from events that we subsequently require. The same principle applies to how we set up the Cosmos DB, because some things you can't change later (at least without some pain).

We know that we will be handling updated and deleted events post MVS, so we should think through if the system we're iteratively building, and also the data we capture, will be able to support known future requirements.

I think in this case though, we're OK to not store any deleted or updated dates, because the system doesn't currently set them, so we won't be losing any data. When a newer version of the system is running, we can interpret their absence as equivalent to null.

Reply:
We do store updated, but we set it on updating the account with a newly created reservation (as opposed to when handling an event that updates an existing reservation). I don't think we have to worry about updated for now, at least for `ReservationCreatedEvent`'s.